### DISCUSSION TITLE
Ko-fi Full

### DISCUSSION THREAD

**Discussion by XMete** [KevinBytesTheDust/CVAmp#272] [Q&A]
Hello, I want to buy via Ko-FI, but it says FULL. Does it also work on kick as current?

**Comment by jlplenio** [comment]
Hey you. Yes, kick works, if you have at least the [recommended medium plan](https://github.com/jlplenio/crude-twitch-viewer-bot/wiki/Webshare.io-Proxies-Guide#--medium-plan-22month). I keep Kofi at a low and fixed number to have the capacity to address all incoming requests and discussions. When people leave, the space automatically becomes free. One is free atm.
